website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+---
+layout: docs
+page_title: Vault identity duplicate resolution guide
+description: >-
+   Discover whether your Vault cluster contains any identity duplicates and if
+   so, how to safely resolve them.
+---
+
+# Vault identity duplicate resolution guide
+
+Some users may have duplicate identity resources (entities, aliases, groups) in
```

Comment:
How about "identity engine resources" with the word "identity engine" linking to the introductory identity docs page, so that people who come to this page from the 1.19 upgrade guide and aren't very familiar with entities/aliases/groups can do some initial reading to remind themselves what these things are first before they continue.